perfkitbenchmarker/benchmarks/fio_benchmark.py

```diff
@@ -298,13 +310,13 @@ def Prepare(benchmark_spec):
       fill_size = FLAGS.device_fill_size
     else:
       fill_path = posixpath.join(mount_point, DEFAULT_TEMP_FILE_NAME)
-      if FLAGS['disk_fill_size'].present:
-        fill_size = FLAGS.disk_fill_size
+      if FLAGS['device_fill_size'].present:
```

Comment:
I don't think the bugs were in the last release. I'd like to keep the bugs in this PR, but I'll make a point of separating distinct changes in future PRs. Sorry for mixing the two this time.